src/sb_sw_lib_tests.c.h

```diff
@@ -1620,6 +1620,65 @@ _Bool sb_test_sw_invalid_scalar(void)
     return 1;
 }
 
+// Test that sweet-b is not vulnerable to the "Psychic Paper" attack.
+// More specifically, verify that for some signature (r, s) if r or s = 0
+// signature verification quits out early.
```

Comment:
```suggestion
// signature verification returns invalid.
```

Verification doesn't actually quit out early in this case; it performs the computation but also returns an invalid result always.

https://github.com/westerndigitalcorporation/sweet-b/blob/28e1176fe3ffd204faf069146c0a0ee541927905/src/sb_sw_lib.c#L1089-L1090

In fact, can you add a comment above those two lines indicating that this test validates this property?